### PR TITLE
Add Airplay 2

### LINKED ISSUE #86: Add AirPlay 2 badge
We should show a small speaker badge to accessories that support AirPlay 2 but not HomeKit directly.

### PR DESCRIPTION
**Pull request by kimdv**
Fixes #86

### CODE REVIEW

File: Sources/App/Models/AddAccessoryAirplaySupport.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//
+//  AddAccessoryAirplaySupport.swift
+//  App
+//
+//  Created by Kim de Vos on 27/07/2018.
+//
+
+import FluentPostgreSQL
+
+struct AddAccessoryAirplaySupport: PostgreSQLMigration {
```
Comment (kimdv) [inline Sources/App/Models/AddAccessoryAirplaySupport.swift]: We could remove this class and manually add it. And then add `builder.field(for: \Accessory.supportsAirplay2)` in the `Accessory.prepare`

Comment (BalestraPatrick) [reply Sources/App/Models/AddAccessoryAirplaySupport.swift]: Yeah, I think it's better if we do it manually on the DB only once and remove this class.

File: Resources/Views/accessory.leaf [hunk 2]
```diff
@@ -52,6 +52,12 @@
                                 <p>This accessory requires a proprietary <a href="/accessories/#(accessory.id)">bridge</a>.</p>
                             </div>
                         }
+                        #if(accessory.supports_airplay_2) {
+                            <div id="accessory-page-feature">
+                                <img src="/images/airplay.png">
+                                <p>This accessory supports Airplay 2</p>
```
Comment (BalestraPatrick) [inline Resources/Views/accessory.leaf]: The official name is AirPlay, with the capital P, so we should change it everywhere so users can better recognize it

File: Sources/App/Models/AddAccessoryAirplaySupport.swift (hunk 1 above)
Comment (kimdv) [reply Sources/App/Models/AddAccessoryAirplaySupport.swift]: I will do that!

File: Resources/Views/accessory.leaf (hunk 2 above)
Comment (kimdv) [reply Resources/Views/accessory.leaf]: Fixed !

**Pull request merged**